mne/io/bti/bti.py

```diff
@@ -894,7 +894,8 @@ def _read_bti_header_pdf(pdf_fname):
 def _read_bti_header(pdf_fname, config_fname, sort_by_ch_name=True):
     """ Read bti PDF header
     """
-    info = _read_bti_header_pdf(pdf_fname)
+
+    info = _read_bti_header_pdf(pdf_fname) if pdf_fname is not None else dict()
```

Comment:
you mean the names? yeah I can see that. Otherwise some things are local and it is ok like that. Not all things are equal.